### DISCUSSION TITLE
Still active?

### DISCUSSION THREAD

**Discussion by I2pRandom** [fnuecke/oc2#277] [General]
Is the Development still active? it's been a while since there's been any commits. 
I did see something about a RISC-V bug.

**Comment by CoolCat467** [comment]
This is another duplicate of https://github.com/fnuecke/oc2/discussions/266 and https://github.com/fnuecke/oc2/discussions/136.
Full response: https://github.com/fnuecke/oc2/issues/253#issuecomment-1336404935

Basically it's on hiatus at the moment because of an issue in the underlying emulator: https://github.com/fnuecke/sedna/issues/12

**Comment by samuelh2005** [comment]
There's https://github.com/North-Western-Development/oc2r, a 1.20.1 development of OC2.
